### ISSUE TITLE
RightClick/Spellcheck stopped working in 0.6.1

### ISSUE THREAD

**Issue by gnozil** [TheGoddessInari/hamsket#239]
The spell check works well in 0.6.0 for Slack App. After upgrading to 0.6.1, the typo was still highlighted, but I cannot right click to call out the suggestion dialog.

	
	-
	> Hamsket 0.6.1
	> Electron 11.0.3
	> linux x64 5.8.0-29-generic
	> 0e58b4f  (grafted, HEAD, tag: nightly, origin/0e58b4f2aeff9d69c6ad7248369097c207911272) Tue, 24 Nov 2020 14:59:19 -0800

**Comment by gnozil** [comment]
It looks like the mouse right click stopped working. E.g., in Office 365 Outlook, I cannot right click on a link to copy the URL.

**Comment by aravindhp** [comment]
I am seeing the same behavior.
```
Version: 0.6.1
Platform: linux (x64)
Electron: 11.0.3
Chromium: 87.0.4280.67
Node: 12.18.3
BuildVersion: 5c9df38 (grafted, HEAD, tag: 0.6.1, origin/5c9df38848a6e2de26c6b7fe4fd6bfbef71ef06d) Fri, 4 Dec 2020 10:53:50 -0800
```

**Comment by TheGoddessInari** [comment]
Update to electron caused this, electron-context-menu and electron-contextmenu-wrapper are both affected, but it's not entirely clear what the cause is. Should I roll back electron until this is dealt with, I assume?

**Comment by gnozil** [comment]
Updated to 0.6.2, still see the same issue.


	-
	> Hamsket 0.6.2
	> Electron 11.0.3
	> linux x64 5.8.0-31-generic
	> 323503b  (grafted, HEAD, tag: nightly, origin/323503be7f2c168b830a3e32a26f6348a587fb3b) Fri, 4 Dec 2020 10:57:55 -0800

**Comment by jsbarde** [comment]
I'm having the same issue as well on 0.6.1.

**Comment by Nadeox1** [comment]
Yep, the issue seems to persists if downgrading from 0.6.2 to 0.6.1.
I guess it does not really downgrade Electron?

**Comment by jsbarde** [comment]
> Yep, the issue seems to persists if downgrading from 0.6.2 to 0.6.1.
> I guess it does not really downgrade Electron?

I have a version of 0.6.0 nightly that I saved that I had tagged as "working spellcheck" and downgrading to it does fix the problem, so I think it's broken in 0.6.1 and 0.6.2 nightly, at least at the moment.

**Comment by Nadeox1** [comment]
Works again in 0.6.2 (non-nightly) :)

**Comment by aravindhp** [comment]
Works with [0.6.2](https://github.com/TheGoddessInari/hamsket/releases/tag/0.6.2)

**Comment by jsbarde** [comment]
I can also confirm fixed with 0.6.2. Thanks @TheGoddessInari

**Comment by TheGoddessInari** [comment]
Sorry about that, that should work the republished 0.6.2, containing GIT commit fd10375.

Let me know of any further issues. Had to downgrade electron back, since it's totally an upstream issue.
